Prolangle/Languages/TypeScript.cs

```diff
@@ -0,0 +1,27 @@
+using Prolangle.Languages.Framework;
+
+namespace Prolangle.Languages;
+
+public class TypeScript : BaseLanguage
+{
+	private TypeScript()
+	{
+	}
+
+	public static TypeScript Instance { get; } = new();
+
+	public override Guid Id { get; } = Guid.NewGuid();
+	public override string Name => "TypeScript";
+	public override TypeSystem Typing => TypeSystem.Structural | TypeSystem.Strong | TypeSystem.Static;
+	public override bool Compiled => true;
+	public override MemoryManagement MemoryManagement => MemoryManagement.TracingGarbageCollection;
+	public override SyntaxStyle SyntaxStyle => SyntaxStyle.C;
+
+	public override Applications KnownForBuilding => Applications.Client | Applications.Server | Applications.Web;
+
+	public override Paradigms Paradigms =>
+		Paradigms.EventDriven | Paradigms.Functional | Paradigms.ObjectOriented | Paradigms.Generic;
+
+	public override double? TiobeRating => 0.37;
```

Comment:
FWIW, it's `0.23` now. But I don't think it's worth chasing these metrics over time _too much_.